src/app/pages/send/ordinal-inscription/coinselect/select-inscription-coins.ts

```diff
@@ -65,9 +66,10 @@ export function selectTaprootInscriptionTransferCoins(
     if (nextUtxo) neededInputs.push(nextUtxo);
     utxos = remainingUtxos;
     txSize = txSizer.calcTxSize({
-      input_script: 'p2tr',
+      input_script: 'p2wpkh',
       input_count: neededInputs.length + 1,
       p2tr_output_count: 1,
+      p2wpkh_output_count: 1,
```

Comment:
there might be not `p2wpkh` output, in case user sends inscription using his whole balance for fees.
but it's a rare case I suppose, we can fix it later

Reply:
Yeah realised this but at least we over-estimate vs under.